blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueTrend.java

```diff
@@ -0,0 +1,15 @@
+package io.jenkins.blueocean.rest.model;
+
+import org.kohsuke.stapler.export.Exported;
+
+public abstract class BlueTrend extends Resource {
+
+    public static final String DATA = "data";
+    public static final String ID = "id";
+
+    @Exported(name = ID)
+    public abstract String getId();
```

Comment:
How is the id used? Is this a unique opaque id or is more of description of trend?

Reply:
It's a "name" or description of the trend, e.g "junit" or "stageDuration"